docs/modeling-your-data/modeling-your-data-with-dbt/dbt-custom-models/index.md

```diff
@@ -31,10 +31,12 @@ The `_this_run` and derived tables are considered part of the 'public' class of
 
 ## What denotes a custom module?
 
-**Does:**  
+**Does:**
```

Comment:
To be honest these are a bit weird grammatically. I’d rather rephrase this entirely.